### PR TITLE
Update 00-hpc-intro.md

### PR DESCRIPTION
**Pull request by mpiercy**
pointing out a few more advantages to HPC, no sys admin duties, researchers can focus on research

### CODE REVIEW

File: _episodes/00-hpc-intro.md [hunk 1]
```diff
@@ -67,6 +67,12 @@ Using a cluster often has the following advantages for researchers:
 * **Convenience.** Maybe your calculations just take a long time to run or are
   otherwise inconvenient to run on your personal computer. There's no need to
   tie up your own computer for hours when you can use someone else's instead.
+  Also, users can focus on their research and not systems administration.  
+  You don't have to worry about security updates, O.S. patching, hardware
+  replacement/failures, networking and systems monitoring. In a typical HPC 
+  cluster these hassles are taken care of by others.  Because of the nature of 
+  the Linux filesystem sharing data/files if easy and can be often managed by
```
Comment (ChristinaLK) [inline _episodes/00-hpc-intro.md]: if -> is

File: _episodes/00-hpc-intro.md [hunk 2]
```diff
@@ -67,6 +67,12 @@ Using a cluster often has the following advantages for researchers:
 * **Convenience.** Maybe your calculations just take a long time to run or are
   otherwise inconvenient to run on your personal computer. There's no need to
   tie up your own computer for hours when you can use someone else's instead.
+  Also, users can focus on their research and not systems administration.  
+  You don't have to worry about security updates, O.S. patching, hardware
```
Comment (psteinb) [inline _episodes/00-hpc-intro.md]: O.S. -> operating system

**Comment by psteinb** [comment]
@mpiercy I didn't see any acitivity on this PR. Can you please have a look at our comments? Thanks.

**Comment by tkphd** [comment]
I do not agree with the statement, since it is not true in the general case.
Clusters span a broad spectrum, including

* Beowulf clusters assembled from spare machines, with low performance but high educational merit
* Research group clusters comprising on the order of a dozen nodes, with better performance due to upgraded interconnect and data servers
* Department clusters with centralized user management, fast interconnect, and robust data management
* University clusters and supercomputers
* National laboratory supercomputers

Administration at each stage of the spectrum grows in complexity. In perhaps the first two, a member of the group can be tasked part-time to cover these duties. Higher up the chain, organizations must hire dedicated teams to the management, maintenance, and administration of the machinery. 

The resources required to maintain a *high-performance* cluster should probably be categorized as a *disadvantage* of using clusters.

Students of  HPC Carpentry should be aware of this escalating support infrastructure. Some will work at a university or national lab, with access to a resource managed at the organization level. But that won't be everyone, and we should teach them about the requirements of building *and maintaining* a performant cluster.

**Comment by psteinb** [comment]
@tkphd I agree and like the conceptional overview. At the same time, I also understand the more user based view that @mpiercy expressed. I see a misconception of the term 'cluster' at the heart of the different views (or maybe 'cluster' is fuzzy in the first place). Depending on the side which you work on, people for sure have different views of the pros/cons of a 'cluster'. I suggest that @mpiercy reacts to our comments and we merge this PR. We should however put your comment from above into [_extras/discuss.md](https://hpc-carpentry.github.io/hpc-intro/guide/index.html) or [_extras/guide.md](https://hpc-carpentry.github.io/hpc-intro/guide/index.html).

I personally don't like this episode in its current state. From a self-learning perspective, the content is overwhelming and does put an avalanche of technical terms to the eyes of a potentially novice reader. In an in-class setting, that might be OK if coupled with activities that help clarify any misunderstandings and level the knowledge to some degree of all learners. Further, it's content is controversial from my point of view ... your comment made that clear to me.

**Comment by ChristinaLK** [comment]
I use cluster as a short-hand for "centrally administered, large-scale, shared, scheduled" computing system but haven't found a short way to be sufficiently general and still clear about what I mean.

**Comment by psteinb** [comment]
this is a fruitful discussion. I would love to ask @mpiercy to pick it up and weave it into `00-hpc-intro.md`. I don't like open PRs lying around and with no action. if there is no activity with regards to contributions to the material and/or adding our comments by end of September 2018, I'd vote to close this PR unmerged.

**Comment by tkphd** [comment]
@psteinb, note that @mpiercy did make corrections yesterday in 975AC96 (thanks Mark!). Are you also requesting that Mark fix our clarify the definition of a cluster? 
Since that feels like PR scope creep, I would be happy to migrate these discussion points into the instructor notes, and am in favor of merging this PR.

**Comment by psteinb** [comment]
@tkphd agreed. @mpiercy can you please only resolve the merge conflicts and then we are good to go.

**Comment by ChristinaLK** [comment]
@tkphd Fair enough. I think we agree on the scope of the lesson covering how to use "any interconnected set of computers able to share a distributed workload" but I'm also assuming that we're training people on a resource they don't manage (they're not admins) + that is scheduled in some way (no direct ssh to nodes except a head node) 

So I think you're right in that the definition is very broad, but I do think there are some "cluster" cases that we are *not* planning to tackle in this lesson.

**Comment by tkphd** [comment]
@psteinb, I went ahead and rebased @mpiercy's work on a new branch: 
[tkphd/hpc-intro:issue23_update-hpc-intro](https://github.com/tkphd/hpc-intro/tree/issue23_update-hpc-intro).
Since @mpiercy worked on `gh-pages` directly, merging my rebased branch back into their repo is fraught. Instead, I would suggest merging my rebased branch and closing this PR.

@mpiercy, your work will still be reflected in the merged result: take a look at my branch's [commit history](https://github.com/tkphd/hpc-intro/commits/issue23_update-hpc-intro), and you'll see that your commits are preserved with proper attribution. They're just moved forward in time, reflecting commits made in `gh-pages` since your original contribution. Thanks for the contribution.

**Comment by psteinb** [comment]
sounds like a good plan and thanks for this demo of your git-fu. I'll close this PR now.

**Comment by tkphd** [comment]
To prevent this from getting lost, I would recommend re-opening this PR (#23) until the rebased branch gets merged (hopefully soon). It should be closed by a [keyword](https://help.github.com/articles/closing-issues-using-keywords/) in the merging commit. Unless you (@psteinb) would prefer that I open a separate PR... but that just seems like clutter.

**Pull request closed without merging**